homeassistant/config_entries.py

```diff
@@ -1937,6 +1937,29 @@ def async_create_entry(  # type: ignore[override]
 
         return result
 
+    @callback
+    def async_update_and_reload_entry(
```

Comment:
We do this all over the place with reauth so I think its useful.

I think its unclear that this will also return `self.async_abort` from the name.

Reply:
Hm what about `async_update_reload_and_finish_entry `?

Reply:
Maybe `async_update_reload_and_abort` shows the intent better since its always an abort and finish can be abort or create